src/API/CompanyName.MyMeetings.API/Modules/Meetings/MeetingCommentingConfiguration/MeetingCommentingConfigurationController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading.Tasks;
+using CompanyName.MyMeetings.API.Configuration.Authorization;
+using CompanyName.MyMeetings.Modules.Meetings.Application.Contracts;
+using CompanyName.MyMeetings.Modules.Meetings.Application.MeetingCommentingConfiguration.DisbaleMeetingCommentingConfiguration;
+using CompanyName.MyMeetings.Modules.Meetings.Application.MeetingCommentingConfiguration.EnableMeetingCommentingConfiguration;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CompanyName.MyMeetings.API.Modules.Meetings.MeetingCommentingConfiguration
+{
+    [Route("api/meetings/meetings/{meetingId}/configuration/commenting")]
+    [ApiController]
+    public class MeetingCommentingConfigurationController : ControllerBase
+    {
+        private readonly IMeetingsModule _meetingsModule;
+
+        public MeetingCommentingConfigurationController(IMeetingsModule meetingsModule)
+        {
+            _meetingsModule = meetingsModule;
+        }
+
+        [HttpPatchAttribute("disable")]
+        [HasPermission(MeetingsPermissions.EditMeetingCommentingConfiguration)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<IActionResult> DisableCommenting(Guid meetingId)
+        {
+            await _meetingsModule.ExecuteCommandAsync(new DisableMeetingCommentingConfigurationCommand(meetingId));
+            return Ok();
+        }
+
+        [HttpPatchAttribute("enable")]
+        [HasPermission(MeetingsPermissions.EditMeetingCommentingConfiguration)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<IActionResult> EnableCommenting(Guid meetingId)
```

Comment:
In my opinion, we should have permissions per action. In this case DisableMeetingCommenting and EnableMeetingCommenting. What do you think?

Reply:
I think you are right. Permission per action gives us maximum flexibility to configure authorization without changing code of controllers. We can do it by managing user's roles and role's permissions. And moreover such granular permissions are more explicit and allows us not to think hard about meaningful permission name :)